LjinuxRoot/bin/jpkg.py

```diff
@@ -0,0 +1,184 @@
+opts2 = ljinux.based.fn.xarg()
+errored = False
+
+jpkg_version = 0
+
+if len(opts2["w"]) > 1 and opts2["w"][0] == "install":
+    # preperation
+
+    fl = []  # folder list
+
+    # load system package list
+    ljinux.based.command.fpexecc([None, "/etc/jpkg/tools/generatelist.py"])
```

Comment:
Make lJinux version of subprocess.run

```
# In lJinux.based.command 
def run(content, script_flag=None) -> None:
      """
       :content -> list of str  example ["mkdir", "-p", "foo/bar"]
       content[0] -> script name "mkdir"
       content[1..N] -> script's flags ["-p", "foo/bar"]
      """
      ljinux.based.user_vars['argj'] = ' '.join(content[1:])
      ljinux.based.command.fpexecc([script_flag, content[0]])
```
And on this line do
`ljinux.based.command.run(["/bin/mkdir", "a " + extpath ])`